### PR TITLE
Added CircleCI Swag Offer For Hacktoberfest

### PR DESCRIPTION
**Pull request by monizb**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by monizb** [comment]
Done @crweiner :)

File: README.md [hunk 1]
```diff
@@ -149,6 +156,13 @@ None right now.
 - **How to sign up**: Make a PR on the [lakeFS Repo](https://github.com/treeverse/lakeFS) and we'll send you a link.
 - **Notes**: Check out our [contributing guide](https://docs.lakefs.io/contributing) and join our [slack channel](https://join.slack.com/t/lakefs/shared_invite/zt-g86mkroy-186GzaxR4xOar1i1Us0bzw) for help and discussions
 
+#### **CircleCI** (T-shirt,stickers and Magic Pin)
+
+- **Requirements**: 1-3 PRs for stickers,4 PRs for a T-Shirt and All the items with the pin if you develop a new Orb from scratch
+- **Swag**: T-shirt, stickers
+- **How to sign up**: Make the contrbutions accordingly and then use [this](https://circleci-community.typeform.com/to/mErGL0nv) form to submit your PRs
+- **Notes**: Join the #Orbtoberfest discussion on our [community forum](https://discuss.circleci.com/t/official-hacktoberfest-and-orbtoberfest-thread/37510) for ideas, FAQs, and to ask any questions. Also check out the official website (here)[https://hacktoberfest.circleci.com/]
```
Comment (crweiner) [inline README.md:164]: Take another peak at the last link in this line, please. You have reversed the link syntax. 😄 It should be ```[here](URL)```
```suggestion
- **Notes**: Join the #Orbtoberfest discussion on our [community forum](https://discuss.circleci.com/t/official-hacktoberfest-and-orbtoberfest-thread/37510) for ideas, FAQs, and to ask any questions. Also check out the official website [here](https://hacktoberfest.circleci.com/)
```

File: README.md [hunk 2]
```diff
@@ -149,6 +156,13 @@ None right now.
 - **How to sign up**: Make a PR on the [lakeFS Repo](https://github.com/treeverse/lakeFS) and we'll send you a link.
 - **Notes**: Check out our [contributing guide](https://docs.lakefs.io/contributing) and join our [slack channel](https://join.slack.com/t/lakefs/shared_invite/zt-g86mkroy-186GzaxR4xOar1i1Us0bzw) for help and discussions
 
+#### **CircleCI** (T-shirt,stickers and Magic Pin)
+
+- **Requirements**: 1-3 PRs for stickers,4 PRs for a T-Shirt and All the items with the pin if you develop a new Orb from scratch
```
Comment (crweiner) [inline README.md:161]: Can you please break this down into separate sub-bullet points like this:
```suggestion
- **Requirements**: 
 - 1-3 PRs: stickers
 - 4 PRs: T-Shirt
 - Develop a new Orb from scratch: All of the items, plus a magic pin
```

**Review by crweiner — COMMENTED** [review]
Hi @monizb! Just a few small changes, please, and then you should be good to go! Thank you! 👍

**Comment by monizb** [comment]
@crweiner hopefully that resolves everything :)

**Comment by crweiner** [comment]
Hey @monizb! Thank you so much for contributing to the 2020 Hacktoberfest Swag List and adding CircleCI for Orbtoberfest! I appreciate your hard work. Have a great Hacktoberfest!

thank you

**Pull request merged**